src/extension/content-script/batteries/Reddit.ts

```diff
@@ -0,0 +1,100 @@
+import getOriginData from "../originData";
+import setLightningData from "../setLightningData";
+
+declare global {
+  interface Window {
+    LBE_REDDIT_MUTATION_OBSERVER: MutationObserver;
+  }
+}
+
+const urlMatcher = /^https:\/\/www.reddit\.com\/user\/(\w+).*/;
+
+// can we get the user description from the primary column that holds the profile information?
+function isOnProfilePage() {
+  return document.querySelector('[class="bVfceI5F_twrnRcVO1328"]') != null;
```

Comment:
Arg, not sure if you got the first comment. Had some network issues.   
We also had a look at this yesterday and were a bit worried that these classnames might change because they look dynamic and this might be a way of avoiding others to scrape reddit. What do you think? Would there be a nicer way/saver to find the wanted divs?

Reply:
Yeah I definitely thought the same thing when I was first looking at their classnames but surprisingly this class is universal for all of the user profile descriptions (I checked multiple profiles)

I couldn't find any other way to programmatically grab this element

Reply:
Interesting. Good to know! I guess it would be ok to start with this and adjust if something changes.

Reply:
My thoughts exactly, this works fine for now but it would be ideal to find a more robust method for grabbing this element over the long term.